api/parse.py

```diff
@@ -101,8 +31,85 @@ def FoodDatabase():
     grain = df["grain"]
     meat = df["meat"]
     fruit_veg = df["fruit_veg"]
-    print(df)
-    return dairy, grain, meat, fruit_veg
+    food_groups = np.concatenate((dairy, meat, grain, fruit_veg))
+    # print(type(food_groups))
+    #return dairy, grain, meat, fruit_veg
+    return food_groups
+
+def nlp():
+    nlp = spacy.load('en_core_web_sm')
```

Comment:
Name this var something different from the function name, prevents confusion from thinking it's recursive.